src/swap/components/SwapMessage.tsx

```diff
@@ -4,12 +4,18 @@ import { getSwapMessage } from '../utils/getSwapMessage';
 import { useSwapContext } from './SwapProvider';
 
 export function SwapMessage({ className }: SwapMessageReact) {
-  const { to, from, error, loading, isTransactionPending } = useSwapContext();
+  const { address, to, from, error, loading, isTransactionPending } =
+    useSwapContext();
+
+  const isMissingRequiredFields =
```

Comment:
On this part, I was refering to.

Reply:
the `statusData` checks here are to get around typescript errors since `isMissingRequiredField` only exists on 2 of the life cycle statuses.

Reply:
For a follow up PR, maybe we need `isMissingRequiredField` in all `statusName`